R/FilterStateRange.R

```diff
@@ -493,6 +526,74 @@ RangeFilterState <- R6::R6Class( # nolint
               }
             }
           )
+          
+          private$observers$selection_manual <- observeEvent(
+            ignoreNULL = FALSE,
+            ignoreInit = TRUE,
+            eventExpr = input$selection_manual,
+            handlerExpr = {
+              # 3 separate checks are required here to prevent errors
+              #
+              # if the user sets either input to 'e' it will return NA
+              # this NA would cause the lower > upper check to return NA
+              #  and the if(lower > upper) check would throw an error
+              #
+              # if lower > manual, contain_interval() will error because it
+              #  expects it's input to be sorted
+              if (any(is.na(input$selection_manual))) {
+                showNotification(
+                  "Numeric range values must be numbers.",
+                  type = "warning"
+                )
+                shinyWidgets::updateNumericRangeInput(
+                  session = session,
+                  inputId = "selection_manual",
+                  value = self$get_selected()
```

Comment:
this is why it fails
```suggestion
                  value = private$get_selected()
```

Reply:
please replace all `self$get_selected` with `private$get_selected` and then module should be robust enough